src/include/txn_inline.h

```diff
@@ -719,6 +719,21 @@ __txn_visible_all_id(WT_SESSION_IMPL *session, uint64_t id)
     return (WT_TXNID_LT(id, oldest_id));
 }
 
+/*
+ * __wt_txn_timestamp_visible_all --
```

Comment:
What do you think of the following name?

```suggestion
 * __wt_txn_timestamp_globally_visible --
```

Reply:
No, we are using `visible_all` in most of the places, as visible_all can be used for both globally visible and obsolete, so I prefer the existing name. I modified the description to be more clear.